### DISCUSSION TITLE
Support nthRoot((-8), (3/2))

### DISCUSSION THREAD

**Discussion by sonnyk22** [josdejong/mathjs#2707] [Ideas]
The library works fantastic when in providing the correct result of `4` for `(-8)^(2/3)`, but it does not return the the same result when `nthRoot(-8, 3/2) or nthRoot((-8), (3/2))`; it returns `Error: Root must be odd when a is negative error` message. I am getting `4`returning from the TI on this as well.  I am using the `number`setting in the `config()`function.  Is there an extra setting that I am missing?

**Comment by josdejong** [comment]
Thanks for reporting. The algorithm used to calculate `nthRoot` has limitations, would be nice if we could improve this.
